src/Organizer/CommandHandler/UpdateContributorsHandler.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+declare(strict_types=1);
+
+namespace CultuurNet\UDB3\Organizer\CommandHandler;
+
+use Broadway\CommandHandling\CommandHandler;
+use CultuurNet\UDB3\Contributor\ContributorRepository;
+use CultuurNet\UDB3\Model\ValueObject\Identity\UUID;
+use CultuurNet\UDB3\Organizer\Commands\UpdateContributors;
+use CultuurNet\UDB3\Organizer\OrganizerRepository;
+
+final class UpdateContributorsHandler implements CommandHandler
+{
+    private OrganizerRepository $organizerRepository;
+
+    private ContributorRepository $contributorRepository;
+
+    public function __construct(OrganizerRepository $organizerRepository, ContributorRepository $contributorRepository)
+    {
+        $this->organizerRepository = $organizerRepository;
+        $this->contributorRepository = $contributorRepository;
+    }
+    public function handle($command): void
```

Comment:
Nit: Usually we put an empty line before each method, but it looks like the linter does not check that

Reply:
I have too much faith in `composer cs-fix` 🫣, I'll fix it.

Reply:
That's okay, that's what is there for! You shouldn't have to check every file manually yourself.